packages/dropdown/src/react/sub-label.tsx

```diff
@@ -1,29 +1,31 @@
-import {
-  names as themeNames,
-  useTheme
-} from '@pluralsight/ps-design-system-theme'
-import { ValueOf } from '@pluralsight/ps-design-system-util'
-import glamorDefault, * as glamorExports from 'glamor'
+import { useTheme } from '@pluralsight/ps-design-system-theme'
+import { HTMLPropsFor, classNames } from '@pluralsight/ps-design-system-util'
 import React from 'react'
 
-import stylesheet from '../css/index'
+import '../css/index.css'
 
-const glamor = glamorDefault || glamorExports
-
-const styles = (themeName: ValueOf<typeof themeNames>) => {
-  const label = 'psds-dropdown__sub-label'
-
-  return glamor.compose(
-    glamor.css(stylesheet[`.${label}`]),
-    glamor.css(stylesheet[`.${label}.psds-theme--${themeName}`])
-  )
+interface SubLabelProps extends HTMLPropsFor<HTMLDivElement> {
+  subLabel?: React.ReactNode
 }
 
-export const SubLabel: React.FC<{ subLabel?: React.ReactNode }> = ({
-  subLabel
+export const SubLabel: React.FC<SubLabelProps> = ({
+  className,
+  subLabel,
+  ...rest
 }) => {
   const themeName = useTheme()
-  return subLabel ? <div {...styles(themeName)}>{subLabel}</div> : null
+  return subLabel ? (
+    <div
+      {...rest}
+      className={classNames(
+        'psds-dropdown__sub-label',
+        `psds-theme--${themeName}`,
+        className
+      )}
```

Comment:
While this one is fine I think inlining them like this makes it easier to miss stuff during the refactor like in some of my other comments.

Reply:
I'm a little unclear on your meaning. You're saying inlining `className` values makes it easier to miss stuff. What stuff? You refer to other comments. Which ones? If I knew which ones, maybe I'd see what you meant by missing stuff. Perhaps you're talking about this one? https://github.com/pluralsight/design-system/pull/1881#discussion_r662915914 If so, what are you thoughts on how not inlining the className value would help not miss a selector?